python/src/response.py

```diff
@@ -46,12 +48,22 @@ def _get_response_msg(self, s3_keys=None):
             "response": {"cacheable": self.cacheable, "error": self.error},
         }
 
+    @xray_recorder.capture('Response._upload')
     def _upload(self, result):
         client = boto3.client("s3", **config.BOTO_RESOURCE_KWARGS)
         key = "{}/{}".format(self.request["uuid"], str(uuid.uuid4()))
         body = result.get_result_object()["body"]
 
+        was_enabled = global_sdk_config.sdk_enabled()
+
+        if was_enabled:
+            global_sdk_config.set_sdk_enabled(False)
+
         client.put_object(Key=key, Bucket=self.s3_bucket, Body=body)
+
+        if was_enabled:
+            global_sdk_config.set_sdk_enabled(True)
+
```

Comment:
Can you explain this part Marcell? It seems like the SDK is disabled before putting the object, and then enabled again after that?

Reply:
Correct, the reason is that there is a bug in botocore's S3.

See: https://github.com/open-telemetry/opentelemetry-python-contrib/issues/298

Reply:
Added a comment re this.